src/hotspot/share/oops/unloadableMethodHandle.inline.hpp

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+#ifndef SHARE_OOPS_UNLOADABLE_METHOD_HANDLE_INLINE_HPP
+#define SHARE_OOPS_UNLOADABLE_METHOD_HANDLE_INLINE_HPP
+
+#include "unloadableMethodHandle.hpp"
+
+#include "memory/universe.hpp"
+#include "oops/klass.inline.hpp"
+#include "oops/oopHandle.inline.hpp"
+#include "oops/weakHandle.inline.hpp"
+
+inline UnloadableMethodHandle::UnloadableMethodHandle(Method* method) {
```

Comment:
This should initialize method in the ctor initializer list.

Reply:
Maybe, but the field is not `const`, so there seem to be no point? We also assign after assert checks `method` for us.